File: in_typescript/src/Pattern_16.ts
export default function pattern_16() {
  let row = 9;
  let column = 9;
  let symbol = "*";
  let index = 0;

  for (let i = 0; i < row; i++) {
    for (let j = 0; j < column; j++) {
      if (j == index) {
        process.stdout.write("\\");
      } else if (j == column - 1 - index) {
        process.stdout.write("/");
      } else {
        process.stdout.write(`${symbol}`);
      }
    }
    index++;
    process.stdout.write("\n");
  }
}

File: in_rust/src/pattern_16.rs
pub fn pattern_16() {
    let row = 9;
    let column = 9;
    let symbol = "*";
    let mut index = 0;

    for _ in 0..row {
        for j in 0..column {
            if j == index {
                print!("\\")
            } else if j == column - 1 - index {
                print!("/")
            } else {
                print!("{}", symbol)
            }
        }
        index += 1;
        println!();
    }
}
